core/src/test/scala/org/apache/spark/network/nio/ConnectionManagerSuite.scala

```diff
@@ -261,7 +261,7 @@ class ConnectionManagerSuite extends FunSuite {
     val clientConf = new SparkConf
     clientConf.set("spark.authenticate", "false")
     val ackTimeout = 30
```

Comment:
You could consider renaming this `acktimeoutS` too, but not that important either way.